### ISSUE TITLE
lint error: use of deprecated item 'std::error::Error::description': use the Display impl or to_string()

### ISSUE THREAD

**Issue by rbtcollins** [rust-lang-nursery/error-chain#286]
This is showing up in stable rust now.
```
use of deprecated item 'std::error::Error::description': use the Display impl or to_string()
```

I still love error-chain very much; perhaps I shouldn't? Is help needed?

**Comment by LukasKalbertodt** [comment]
Fixed by #285, but not yet released.

**Comment by AndyGauge** [comment]
[12.2](https://crates.io/crates/error-chain) Addresses this issue.
